tests/conftest.py

```diff
@@ -2,6 +2,7 @@
 from functools import wraps
 
 import pytest
+from eth.codecs.abi import encode
```

Comment:
I'd prefer to see these imported qualified, so that the usage looks like abi.encode / abi.decode.